docker-compose.yml

```diff
@@ -3,8 +3,7 @@ version: '3.7'
 services:
 
   cheshire-cat-core:
-    build:
-      context: ./core
+    image: ghcr.io/cheshire-cat-ai/core:latest
```

Comment:
The already built image is meant to be used outside of this repo:
- if you want to `develop` on cat core, you git pull the code and build the image locally (in this case the image opens a volume on the code which is downloaded and versioned via git)
- if you want to just `use` the cat, you directly run the ghcr.io docker image, opening a volume just on the plugin folder and eventually some other folders

I don't know if that is clear, otherwise I'll reword :)

Reply:
As discussed privately, the file is ok as it is